.github/workflows/push.yml

```diff
@@ -14,6 +14,8 @@ jobs:
         with:
           architecture: x64
           version: ${{ matrix.racket-version }}
-      - run: raco pkg install --batch --auto --name rackcheck
-      - run: raco test .
-      - run: raco test examples/*
+      - run: raco pkg install --batch --auto ./rackcheck-lib
+      - run: raco pkg install --batch --auto ./rackcheck
+      - run: raco test rackcheck-lib
+      - run: raco test rackcheck/examples/*
+      - run: raco scribble rackcheck/rackcheck.scrbl
```

Comment:
This line isn't necessary since `raco pkg install` already builds the docs.

Reply:
Removed command to raco scribble https://github.com/Bogdanp/rackcheck/pull/10/commits/1d829d68fd374dec11a4001d9cdb3f9943492215